lib/rig_api/v3/apis_test.exs

```diff
@@ -369,6 +369,8 @@ defmodule RigApi.V3.APIsTest do
                  %{"id" => "must have a length of at least 1"},
                  %{"path" => "must be string"},
                  %{"path" => "must have a length of at least 1"},
+                 %{"path_regex" => "must be string"},
+                 %{"path_regex" => "must have a length of at least 1"},
```

Comment:
not sure, but this looks like both `path` and `path_regex` are required to be there and non-empty, while in fact exactly one of them must be present.